spec/schemas/flow_framework.common.yaml

```diff
@@ -77,29 +77,10 @@ components:
           $ref: '#/components/schemas/version'
       minProperties: 1
     FlowFrameworkDeleteResponse:
-      type: object
-      properties:
-        _index:
-          type: string
-          description: The name of the index where the document was stored.
-        _id:
-          type: string
-          description: The unique identifier of the deleted document.
-        _version:
-          type: integer
-          description: The version of the document after the deletion.
-        result:
-          type: string
-          description: The result of the deletion operation.
-          enum: [deleted, not_found]
-        _shards:
-          $ref: '#/components/schemas/shards'
-        _seq_no:
-          type: integer
-          description: The sequence number assigned to the document after the deletion.
-        _primary_term:
-          type: integer
-          description: The primary term assigned to the document after the deletion.
+      allOf:
+        - $ref: '_common.yaml#/components/schemas/WriteResponseBase'
+        - type: object
+          properties: {}
```

Comment:
How do this (`allOff` of WriteResponseBase and an empty object) differ from just referencing `WriteResponseBase` itself?

Reply:
Otherwise LGTM

Reply:
Have simplified it